### ISSUE TITLE
[CoE Starter Kit - BUG] Apps Shared with on Power Platform Admin View Number shows a number but missing the list of people

### ISSUE THREAD

**Issue by mdphung** [microsoft/coe-starter-kit#4966]
### Describe the issue

When I review how many users an app is shared with it shows me a number but there is no one in the list that the app is being shared with on the Power Platform Admin View. Here is a screenshot.

![image](https://user-images.githubusercontent.com/117682988/224736252-1d8d1f2d-7888-46e8-9478-9c30021f041e.png)

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

4.3

### What app or flow are you having the issue with?

Cleanup - Admin | Sync Template V3 (Power Apps Shared with)

### Steps To Reproduce

I have it scheduled to run every other Sunday but I went ahead and ran it a few days ago and a number shows that an app is shared with a number of users. But there is no list of who the app is shared with.

**Comment by Jenefer-Monroe** [comment]
I do not repro this. Can you please go view who the app is shared with? You can see this by clicking Manage Permissions
![image](https://user-images.githubusercontent.com/89584709/224819598-9f9dd765-c4c3-4f75-b6c6-b0845ed9d522.png)

**Comment by mdphung** [comment]
Ok after I added a owner to the app this same app now updates to 0 and it used to say three based on the screenshot I shared earlier. I have Apps Shared with list 1 person listed. but the top highlighted is showing 0.
![image](https://user-images.githubusercontent.com/117682988/225023042-4fc928a1-bbe3-4316-9812-dce84482ff81.png)

**Comment by mdphung** [comment]
I will rerun that again and see if that helps.

**Comment by mdphung** [comment]
Ok so after I ran just the flow it wouldn't update. But I did find something so I noticed the ones that don't have an owner they were giving the weird count even though nothing was shared by looking at managed access or the list on the screen above. So once I added an owner to the app and then reran the Cleanup - Admin | Sync Template V3 (Power Apps Shared With) flow it then updated that screen to the correct numbers.
